dask/dataframe/groupby.py

```diff
@@ -231,12 +230,28 @@ def chunk(df, index):
                 grouped = pd.concat([df, index], axis=1).drop_duplicates()
             return grouped
 
+        key = self.key
+        is_series = isinstance(self.df, Series)
+
         def agg(df):
-            if isinstance(self.df, Series):
+            if is_series:
                 return df.groupby(df.columns[1])[df.columns[0]].nunique()
             else:
-                return df.groupby(level=0)[self.key].nunique()
+                return df.groupby(level=0)[key].nunique()
 
-        return aca([self.df, self.index],
+        return aca([self.df, self.index, self.key],
                    chunk=chunk, aggregate=agg, columns=self.key,
                    token='series-groupby-nunique')
+
+
+def _sum(x):
+    return x.sum()
```

Comment:
Yeah, unfortunately `methodcaller` doesn't serialize well.

```python
In [1]: import numpy as np

In [2]: from cloudpickle import dumps, loads

In [3]: from operator import methodcaller

In [4]: f = methodcaller('sum')

In [5]: x = np.ones((2, 2))

In [6]: f(x)
Out[6]: 4.0

In [7]: loads(dumps(f))(x)
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-7-6f39faa8a209> in <module>()
----> 1 loads(dumps(f))(x)

TypeError: methodcaller needs at least one argument, the method name
```

General rule of thumb, if its in the standard library, it doesn't serialize well.